README.md

```diff
@@ -1,31 +1,45 @@
-| [cypress/base](https://hub.docker.com/r/cypress/base/)         | `cypress/base:16.13.0`      | All operating system dependencies, no Cypress, and no browsers.                    | [![Docker Pulls](https://img.shields.io/docker/pulls/cypress/base.svg?maxAge=604800)](https://hub.docker.com/r/cypress/base/)         |
-| [cypress/browsers](https://hub.docker.com/r/cypress/browsers/) | `cypress/browsers:chrome69` | All operating system dependencies, no Cypress, and some browsers.                               | [![Docker Pulls](https://img.shields.io/docker/pulls/cypress/browsers.svg?maxAge=604800)](https://hub.docker.com/r/cypress/browsers/) |
-| [cypress/included](https://hub.docker.com/r/cypress/included/) | `cypress/included:9.4.1`    | All operating system dependencies, Cypress, and some browsers installed globally.  | [![Docker Pulls](https://img.shields.io/docker/pulls/cypress/included.svg?maxAge=604800)](https://hub.docker.com/r/cypress/included/) |
+If no tag is specified, for example `cypress/included`, then the tag `latest` is used by default: `cypress/included:latest`. It is however recommended to use a specific image tag to avoid breaking changes when new images are released, especially when they include new major versions of Node.js or Cypress.
 
-Of these images, we provide multiple tags for various operating systems and specific browser versions. These allow you to target specific combinations you need.
+Some examples with specific tags including an explanation of the tag meanings are:
 
-## Best practice
+- [cypress/base:18.16.0](https://hub.docker.com/layers/cypress/base/18.16.0/images/sha256-d00c441748e2f1b79d4002bddafe6628f9f9f5458a8a3c66697e622600dc5ad5)
+    Node.js `18.16.0`
+- [cypress/browsers:node-18.16.0-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1](https://hub.docker.com/layers/cypress/browsers/node-18.16.0-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1/images/sha256-e4c1a47c8107c37ca47398d8936743965d871c7285f58b852d5cb2658c400922)
+    Node.js `18.16.0`
+    Chrome `114.0.5735.133-1`
+    Firefox `114.0.2`
+    Edge `114.0.1823.51-1`
+- [cypress/included:12.17.1](https://hub.docker.com/layers/cypress/included/12.17.1/images/sha256-5d541ff206ed28631e720f8fe98dcadf5c62f8e194c028715fb748e564c8c0cc)
+    Cypress `12.17.1`
 
-It is recommended to use a specific image tag, and not rely on the `default` tag. For example, it is better to use `cypress/base:12` than `cypress/base`. Even better it is to use full version of the image, like `cypress/base:12.18.0` - we will never overwrite the existing Docker images to prevent accidental changes.
+Once an image has been published to [Cypress on Docker Hub](https://hub.docker.com/u/cypress) it is frozen. We will never overwrite the existing Docker images to prevent accidental changes.
```

Comment:
We might want to mention that the 'latest' tag will be overwritten

Reply:
Good point. I will try to describe this.

Reply:
looks good to me! Thanks!

Reply:
That's a relief you think so! 🙂 I struggled a bit with the text.